resources/js/Pages/Asisten.vue

```diff
@@ -576,6 +607,61 @@ export default {
         this.menuAllLaporan = $bool;  
     },
 
+    editDescription: function($bool){
+      if ($bool==true) {
```

Comment:
Sorry, i believe this is the last one (i missed it before), when you check a boolean you can just do this one instead :
```js
if ($bool) {
}
```

this is nothing but to remove a redundant check in the code

Reply:
ooh yaa i forget that's bool don't need condition like equal to again sorryyy 🙏😂

Reply:
Haha yeah, its okay, it is easy to forget things like this